Corona-Warn-App/src/main/res/values-night/colors.xml

```diff
@@ -93,5 +93,7 @@
     <color name="contactDiaryOverviewBorderColor">#434445</color>
     <color name="maskGStatusBadgeColor">#31A056</color>
     <color name="maskBadgeColor">#FFFFFF</color>
+    <color name="maskBadgeTextColorRegular">#FFFFFF</color>
+    <color name="maskBadgeTextColorGreenBg">#FFFFFF</color>
```

Comment:
no need to add it to dark file since it is the same as light (default )maskBadgeTextColorGreenBg

Reply:
Removed redundant color